position[] controlPoints
moveConfiguration moveConfiguration
---
bool res
